### PR TITLE
doc: update table to show v0.10 EOL and v7 N/A

### PR DESCRIPTION
**Pull request by jasnell**
Update the table to reflect the current status of v0.10 and v7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,11 @@
   <td>2018-04-18</td>
   <td>2019-04-18</td>
 </tr>
+<tr>
+  <td>N/A</td>
```
Comment (addaleax) [inline README.md]: I know this is a LTS-specific page, but I would really be in favour of writing `Current` here. It makes distinguishing v5’s status from v7’s easier + it’s what the schedule diagram below says.

**Comment by jasnell** [comment]
The collumn is specifically showing LTS status. Putting Current would be a
bit confusing.

**Comment by addaleax** [comment]
Yeah, I was afraid you’d say that. :smile:

**Comment by TheAlphaNerd** [comment]
LGTM

**Pull request merged**